> Hi, can you verify that this works in IE11, as IE8 is too old at this time? Thanks.

Yes, I confirmed that this issue is also present in IE11, and this fixes it.

> source/NVDAObjects/IAccessible/MSHTML.py:333:44: E228 missing whitespace around modulo operator

I'll fix this and try running a local flake8.

--- a/source/NVDAObjects/IAccessible/MSHTML.py
+++ b/source/NVDAObjects/IAccessible/MSHTML.py
@@ -330,7 +330,7 @@ class MSHTMLTextInfo(textInfos.TextInfo):
 			else:
 				raise TypeError("Bookmark was for %s type, not for %s type"%(position.infoClass.__name__,self.__class__.__name__))
 		else:
-			raise NotImplementedError("position: %s"%position)
+			raise NotImplementedError("position: %s" % (position,))
 
 	def expand(self,unit):
 		if unit==textInfos.UNIT_PARAGRAPH:

